ethrpc/methods.go

```diff
@@ -282,6 +283,55 @@ func PendingTransactionCount() CallBuilder[uint] {
 	}
 }
 
+func ContractQuery(contractAddress common.Address, inputAbiExpr, outputAbiExpr string, args interface{}) (CallBuilder[[]string], error) {
```

Comment:
returning an error here, while all other CallBuilder functions here for example Line 335 dont return an error

Reply:
oh thats completely fine, all good :)